src/tests/class_tests/openms/source/AbsoluteQuantitationMethodFile_test.cpp

```diff
@@ -284,8 +217,13 @@ START_SECTION(void store(const String & filename, const std::vector<AbsoluteQuan
     const Param& tm_params1 = aqm_list1[i].getTransformationModelParams();
     const Param& tm_params2 = aqm_list2[i].getTransformationModelParams();
     TEST_EQUAL(tm_params1.size(), tm_params2.size())
-    TEST_REAL_SIMILAR(tm_params1.getValue("slope"), tm_params2.getValue("slope"))
```

Comment:
It is tough to tell from the diff, but does this test for the initial problem that we were discussing?

Reply:
Given headers eg. `slope,intercept` etc (imagine the full names, not just the suffixes), the code does not assume that the cell will contain data. Instead, it uses a default value for the given member's type. This was the original problem.
I added some new lines in the .csv file (1 line misses a param's value, and others miss other information in other columns) to make sure no crash happens now.